test/e2e/adot_test.go

```diff
@@ -0,0 +1,33 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"testing"
+
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+func TestCPackagesAdotInstallSimpleFlow(t *testing.T) {
+	framework.CheckCuratedPackagesCredentials(t)
+	test := framework.NewClusterE2ETest(t, framework.NewDocker(t),
+		framework.WithPackageConfig(t, EksaPackageBundleURI,
+			EksaPackageControllerHelmChartName, EksaPackageControllerHelmURI,
+			EksaPackageControllerHelmVersion, EksaPackageControllerHelmValues),
+	)
+	runAdotInstallSimpleFlow(test) // other args as necessary
+}
+
+func runAdotInstallSimpleFlow(test *framework.ClusterE2ETest) {
```

Comment:
The main test here for adot is to check if metrics are available for scrapping at `8888/metrics`, and the main logic for that is covered under `test/framework/cluster.go`.

I will still call this SimpleFlow since there are more advanced cases I can think so, but I'm still assessing if they are worth adding.